app/services/moderator/merge_user.rb

```diff
@@ -1,5 +1,17 @@
 module Moderator
   class MergeUser < ManageActivityAndRoles
+    MULTIPLE_IDENTITIES_ERROR_MSG = <<~HEREDOC.freeze
+      The user being deleted already has two or more authentication methods.
+      "Are you sure this is the right user to be deleted?
+      "If so, a super admin will need to do this from the console to be safe.
+    HEREDOC
+
+    DUPLICATE_IDENTITIES_ERROR_MSG = <<~HEREDOC.freeze
+      "The user has duplicate authentication methods on the two accounts.
+      "Please double check and delete the identity that the user is no longer using.
+      "This is usually the already-deleted social account, or the authentication method tied to the user to be deleted.
+    HEREDOC
```

Comment:
I wonder if we should start moving these kinds of constants to the translation file with a manageable namespacing strategy, might be out of the scope for this PR